systemd/systemd.go

```diff
@@ -599,7 +599,7 @@ func (s *systemd) getUnitStatus(properties []string, unitNames []string) ([]*Uni
 	// ask for all properties, regardless of unit type
 	cmd[1] = "--property=" + strings.Join(properties, ",")
 	copy(cmd[2:], unitNames)
-	bs, err := s.systemctl(cmd...)
+	multiLineReply, err := s.systemctl(cmd...)
```

Comment:
Hm, in an ideal world this would be it's own PR, AIUI it's making things more readable (and improve the error reporting) without actually changing the functionality? I understand the irony here given that I asked for this. 

I would also make sure the commit does not just say " Michael Review Improvements " but instead describe these changes in some more detail so that our future selfs when doing a `git log` or `git blame` will be able to find some more details without having to go back to the original PR.

Reply:
Talking to Fred OOB he told me that this stuff will go away once https://github.com/snapcore/snapd/pull/11340 is merged.